components/layout/SectionContainer.tsx

```diff
@@ -11,7 +11,8 @@ export default function SectionContainer({
       component="section"
       id={id}
       maxWidth="lg"
-      sx={{ my: 8, scrollMarginTop: 64 }}
+      disableGutters
+      sx={{ my: 8, px: 3, scrollMarginTop: 64 }}
```

Comment:
Would it be easier to consolidate the styling here like you did in get-started.tsx? Consolidating styling here could mean less lines of code in get-started.tsx and other pages.

Reply:
@sramalho94 I'm not 100% sure what you mean, do you mean make the styles a variable?

Reply:
@e-papanicolas yup, seems like there's overlap between this styling and the sectionContainerSxProps object on ln 20 in pages/get-started.tsx.